website/client/config/storybook/mock.data.js

```diff
@@ -72,4 +72,11 @@ export const userStyles = {
     maxHealth: 50,
     maxMP: 158,
   },
+  profile: {
+    name: 'user',
+  },
+  _id: '1337',
```

Comment:
Cute, but shouldn't this be a UUID? (maybe it doesn't matter outside the API...)